test/functionalTest/cases/950_GET_v2_entity/GET_v2_entity_not_found.test

```diff
@@ -0,0 +1,277 @@
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+GET Entity V2 Not found
+--SHELL-INIT--
+dbInit CB
+brokerStart CB
+
+--SHELL--
+
+#
+# 01. GET /v2/entities
+# 02. POST /v1/updateContext
+# 03. GET /v2/entities
+#
+
+echo "01. GET /v2/entities"
+echo "===================="
+orionCurl --url /v2/entities/E1 --json
+echo
+echo
+
+
+echo "02. POST /v1/updateContext"
+echo "=========================="
+payload='{
+  "contextElements": [
+    {
+      "id":   "E1",
+      "type": "T1",
+      "attributes" : [
+        {
+          "name" : "A0",
+          "type": "number",
+          "value" : 255
+        },
+        {
+          "name" : "A1",
+          "value" : "E1/T1/A1"
+        },
+        {
+          "name": "A2",
+          "value": "E1/T1/A2",
+          "type": "AT2"
+        },
+        {
+          "name": "A3",
+          "type": "a",
+          "value": "E1/T1/A3/M1",
+          "metadatas": [
+            {
+              "name": "m1",
+              "value": "M1"
+            }
+          ]
+        },
+        {
+          "name": "A4",
+          "type": "a4",
+          "value": "E1/T1/A4/M1-3",
+          "metadatas": [
+            {
+              "name": "m1",
+              "value": "M1"
+            },
+            {
+              "name": "m2",
+              "type": "tm2",
+              "value": "M2"
+            },
+            {
+              "name": "m3",
+              "type": "number",
+              "value": 3
+            },
+            {
+              "name": "type",
+              "value": "TYPE"
+            },
+            {
+              "name": "value",
+              "value": "VALUE"
+            }
+          ]
+        },
+        {
+          "name": "id",
+          "value": "ID"
+        },
+        {
+          "name": "type",
+          "type": "VALUE",
+          "value": "TYPE"
+        }
+      ]
+    }
+  ],
+  "updateAction": "APPEND"
+}'
+orionCurl --url /v1/updateContext --payload "$payload" --json
+echo
+echo
+
+
+echo "03. GET /v2/entities"
+echo "===================="
+orionCurl --url /v2/entities/E1 --json
+echo
+echo
+
+
+--REGEXPECT--
+01. GET /v2/entities
+====================
+HTTP/1.1 404 Not Found
+Content-Length: 36
+Content-Type: application/json
+Date: REGEX(.*)
+
+{
```

Comment:
If you refer to .test step description, fixed in c59b819. If you refer to description field in error message, it is being discussed in another comment thread.